### PR TITLE
feat: add member entry

### PR DESCRIPTION
**Pull request by adeleke5140**
Add entry json file to member directory.

### CODE REVIEW

File: data/members/Kehinde Adeleke.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "date": "2023-07-14T00:00",
+  "name": "Kehinde Adeleke",
+  "url": "https://www.kehinde.xyz/",
+  "employment": {
+    "hiring": false,
+    "seeking": true
+  },
+  "linkedin": "https://linkedin.com/in/adeleke5140",
+  "mastodon": "https://github.com/adeleke5140",
```
Comment (ericwbailey) [inline data/members/Kehinde Adeleke.json:10]:
```suggestion
  "mastodon": null,
```

Looks like we're using a GitHub URL for a non-GitHub related key here. I raised https://github.com/ericwbailey/a11y-webring.club/issues/120 and self-assigned it, so we could totally do a followup PR to add a GitHub profile URL once there is a proper place for it.

**Review by ericwbailey — CHANGES_REQUESTED** [review]

File: data/members/Kehinde Adeleke.json (hunk 1 above)
Comment (adeleke5140) [reply data/members/Kehinde Adeleke.json]: Thank you for noting the error, Eric. I checked the `json` file multiple times but still missed it.

**Comment by adeleke5140** [comment]
Hi @ericwbailey I took a look at the repo and saw that even for issues self-assigned,  your issue description are well detailed. It really helped me understand what was happening. What is your motivation behind writing self-descriptive issue messages?

**Review by ericwbailey — APPROVED** [review]

File: data/members/Kehinde Adeleke.json (hunk 1 above)
Comment (ericwbailey) [reply data/members/Kehinde Adeleke.json]: No problem! These things happen 🙂

**Comment by ericwbailey** [comment]
First off, welcome aboard! Second, to answer your question (and thank you for asking):

>What is your motivation behind writing self-descriptive issue messages?

I have my attention pulled in a lot of different places these days, so descriptive issues with a lot of cross-linking helps me preserve context for when I come back to them at a later date. Wish there was a better answer than that, but my memory is not that reliable these days.

**Comment by adeleke5140** [comment]
Thank you, Eric.

Thanks for responding. It is very effective I would add. I would be working on making such descriptive PR messages.

**Pull request merged**